### DISCUSSION TITLE
Feature req: support uds: socks5://unix:/path/s.sock

### DISCUSSION THREAD

**Discussion by ghost** [xjasonlyu/tun2socks#67] [Ideas]
Software such as tor offers an option to start socks5 server as a unix domain socket (option socks unix:/path/to/s.sock)
Would be nice is tun2socks was able to talk to it.

Currently, to use tun2socks with uds you need intermediary socat to redirect uds to localhost:port.

less pressing, but vsock (virtio-vsock for VMs) support would also be nice. https://pkg.go.dev/github.com/mdlayher/vsock

**Comment by xjasonlyu** [comment]
Supported
